composer.json

```diff
@@ -1,10 +1,25 @@
 {
-    "require-dev": {
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.4",
-        "wimg/php-compatibility": "dev-master",
-        "wp-coding-standards/wpcs": "dev-develop"
-    },
-    "scripts": {
-        "lint": "phpcs"
-    }
+  "name": "wordpress/gutenberg",
+  "type": "wordpress-plugin",
+  "license": "GPL-2.0",
+  "description": "Prototyping since 1440. Development hub for the editor focus in core.",
+  "homepage": "https://wordpress.github.io/gutenberg/",
+  "keywords": [
+    "gutenberg", "wordpress", "editor", "wp", "react", "javascript"
+  ],
+  "support": {
+    "issues": "https://github.com/WordPress/gutenberg/issues"
+  },
+  "require-dev": {
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.4",
+    "squizlabs/php_codesniffer": "2.9.x",
```

Comment:
What was the reason for staying on the 2.9.x branch?

FWIW, the recent patch that fixed a load of code standards in WP core relied on 3.2.0-dev (`master`) for PHPCS.

WPCS, PHPCompatibility and the DealerDirect plugin work fine with the PHPCS 3.1.1 (stable release).